mopro-ffi/build.rs

```diff
@@ -1,6 +1,28 @@
 fn main() {
     #[cfg(feature = "circom")]
-    if std::env::var("MOPRO_FFI_LINK_TEST_WITNESS").unwrap_or_default() != "" {
-        rust_witness::transpile::transpile_wasm("../test-vectors/circom".to_string());
+    {
+        if std::env::var("MOPRO_FFI_LINK_TEST_WITNESS").unwrap_or_default() != "" {
+            rust_witness::transpile::transpile_wasm("../test-vectors/circom".to_string());
+        }
+
+        // witnesscalc only: copy .dat files to the target folder
```

Comment:
Not able to call `witnesscalc_adapter::build_and_link()`. It copied .dat files to "mopro/target/debug/build/`circom-prover`-c49e075cfd0dba6d/out/witnesscalc/src/multiplier2.dat", but we need .dat files under "...debug/build/`mopro-ffi`-.../src/".

Reply:
created https://github.com/zkmopro/mopro/issues/345  to follow up.